hugegraph-llm/src/hugegraph_llm/demo/rag_web_demo.py

```diff
@@ -542,15 +509,15 @@ def reranker_settings(reranker_type):
         gr.Markdown("""## 2. RAG with HugeGraph 📖""")
         with gr.Row():
             with gr.Column(scale=2):
-                inp = gr.Textbox(value="Tell me about Sarah.", label="Question", show_copy_button=True)
+                inp = gr.Textbox(value=settings.question, label="Question", show_copy_button=True)
```

Comment:
Perhaps we need to standardize the input format here?
If `lines=2` is set to `answer_prompt_input`, `question` also needs it?

Reply:
I agree. The user's question should also be able to support multi-line input, so I will make a simple change🫡